ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario/CreateAndExerciseCommand.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.daml.http.perf.scenario
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+class CreateAndExerciseCommand extends Simulation {
+
+  private val httpProtocol = http.disableWarmUp
+    .baseUrl("http://localhost:7575")
+    .inferHtmlResources()
+    .acceptHeader("*/*")
+    .acceptEncodingHeader("gzip, deflate")
+    .authorizationHeader(
+      "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU")
```

Comment:
Maybe add a comment on what is in this token or even better don’t hardcode it but instead use the jwt encoding functions?

Reply:
I looked into how to make it configurable, not obvious... Gatling uses reflection to instantiate scenario:
```
$ ./bin/gatling.sh --simulations-folder=/home/leos/Projects/daml/ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario --simulation=com.daml.http.perf.scenario.SyncQueryVariableAcs
```

There is no obvious way to pass something via constructor... need to read it from an external config file.

Reply:
Sorry should have made that clearer. I don’t mind hardcoding. I just would like to hardcode the decoded JWT ideally so it’s easy to see which claims it has without having to decode it manually. I’m also fine just adding a a comment along the lines of `actAs: ["Alice"]`